hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -65,40 +75,130 @@ The physical controller supports the following features
 *  Data and informational paritions within each bank of flash memory
 *  Arbitration between host requests and controller requests at the bank level
    *  Host requests are always favored, however the controller priority can escalate if it repeatedly loses arbitration
-   *  Since banks are arbitrated independently, where transactions may take different amounts of times to complete, the physical controller is also responsible for ensuring in-order response to both the controller and host.
+   *  Since banks are arbitrated independently and transactions may take different amounts of times to complete, the physical controller is also responsible for ensuring in-order response to both the controller and host.
 *  Flash read stage
-   *  Each bank maintains a parameterizable number of read buffers in front of the flash memory
+   *  Each bank maintains a parameterizable number of read buffers in front of the flash memory (default to 4)
    *  The read buffers behave as miniature read-only-caches to store flash data when flash words are greater than bus words.
+   *  When a program or erase collides with an entry stored in the read buffer, the buffer contents are invalidated.
```

Comment:
Program buffer invalidated as well?

Reply:
Ignore the above question - I slightly misunderstood when I first read it. 
What happens to reads via the host interface at the then when the scramble cfg changes?

Reply:
Ignore second question as well - It is answered in a later section.